contract_compassion/tests/test_base_module.py

```diff
@@ -0,0 +1,102 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Albert SHENOUDA <albert.shenouda@efrei.net>
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+
+
+from openerp.addons.recurring_contract.tests.test_base_contract \
+    import test_base_contract
+import logging
+logger = logging.getLogger(__name__)
+
+
+class test_base_module(test_base_contract):
+
+    def setUp(self):
+        # Retrieve of income account

```

Comment:
because it will run recurring_contract test in compassion module and it will do the same as contract_compassion when i tryied before